src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+
+import { Todo } from '../types/Todo';
+import { ErrorMessages } from '../types/ErrorNessages';
+
+type Props = {
+  isTodoShow: boolean,
+  newTodoTitle: string,
+  handleNewTodoTitle: (
+    newTodoTitle: React.ChangeEvent<HTMLInputElement>
+  ) => void,
+  addNewTodo: (title: string) => Promise<Todo | null>,
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  isTodoShow,
+  newTodoTitle,
+  handleNewTodoTitle,
+  addNewTodo,
+}) => {
+  const [todoAdd, setTodoAdd] = useState(false);
```

Comment:
The name is a bit confusing. It is not clear that it's boolean
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig